### PR TITLE
Revise notes about Raft Engine

### PR DESCRIPTION
**Pull request by en-jin19**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Revised two notes about Raft Engine

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/8787
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1319,9 +1319,9 @@ Configuration items related to `raftdb`
 Configuration items related to Raft Engine.
 
 > **Note:**
-> 
-> - When you enable Raft Engine for the first time, you need to wait tens of seconds because TiKV switches to the data format of Raft Engine from the one of RocksDB.
-> - The data format of Raft Engine in TiDB v5.4.0 is not compatible with previous versions. Therefore, when downgrading a TiDB cluster from v5.4.0 to earlier versions, you need to disable Raft Engine (by setting the value of `enable` to `false`) **before** downgrading and restart TiKV to make the configuration take effect. Otherwise, you cannot start the downgraded cluster.
+>
+> - When you enable Raft Engine for the first time, TiKV transfers its original data from RocksDB to Raft Engine. Therefore, you need to wait an extra tens of seconds to start TiKV.
+> - The data format of Raft Engine in TiDB v5.4.0 is not compatible with previous versions. Therefore, when you need to downgrade a TiDB cluster from v5.4.0 to earlier versions, **before** downgrading, disable Raft Engine by setting `enable` to `false` and restarting TiKV for the configuration to take effect.
```
Comment (en-jin19) [inline tikv-configuration-file.md:1324]:
```suggestion
> - The data format of Raft Engine in TiDB v5.4.0 is not compatible with previous versions. Therefore, if you need to downgrade a TiDB cluster from v5.4.0 to earlier versions, **before** downgrading, disable Raft Engine by setting `enable` to `false` and restarting TiKV for the configuration to take effect.
```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1319,9 +1319,9 @@ Configuration items related to `raftdb`
 Configuration items related to Raft Engine.
 
 > **Note:**
-> 
-> - When you enable Raft Engine for the first time, you need to wait tens of seconds because TiKV switches to the data format of Raft Engine from the one of RocksDB.
-> - The data format of Raft Engine in TiDB v5.4.0 is not compatible with previous versions. Therefore, when downgrading a TiDB cluster from v5.4.0 to earlier versions, you need to disable Raft Engine (by setting the value of `enable` to `false`) **before** downgrading and restart TiKV to make the configuration take effect. Otherwise, you cannot start the downgraded cluster.
+>
+> - When you enable Raft Engine for the first time, TiKV transfers its original data from RocksDB to Raft Engine. Therefore, you need to wait an extra tens of seconds to start TiKV.
```
Comment (en-jin19) [inline tikv-configuration-file.md:1323]:
```suggestion
> - When you enable Raft Engine for the first time, TiKV transfers its data from RocksDB to Raft Engine. Therefore, you need to wait extra tens of seconds to start TiKV.
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -1319,9 +1319,9 @@ Configuration items related to `raftdb`
 Configuration items related to Raft Engine.
 
 > **Note:**
-> 
-> - When you enable Raft Engine for the first time, you need to wait tens of seconds because TiKV switches to the data format of Raft Engine from the one of RocksDB.
-> - The data format of Raft Engine in TiDB v5.4.0 is not compatible with previous versions. Therefore, when downgrading a TiDB cluster from v5.4.0 to earlier versions, you need to disable Raft Engine (by setting the value of `enable` to `false`) **before** downgrading and restart TiKV to make the configuration take effect. Otherwise, you cannot start the downgraded cluster.
+>
+> - When you enable Raft Engine for the first time, TiKV transfers its data from RocksDB to Raft Engine. Therefore, you need to wait extra tens of seconds to start TiKV.
```
Comment (tabokie) [inline tikv-configuration-file.md:1323]:
```suggestion
> - When you enable Raft Engine for the first time, TiKV transfers its data from RocksDB to Raft Engine. Therefore, you need to wait extra tens of seconds for TiKV to start.
```

**Review by tabokie — APPROVED** [review]

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -1319,9 +1319,9 @@ Configuration items related to `raftdb`
 Configuration items related to Raft Engine.
 
 > **Note:**
-> 
-> - When you enable Raft Engine for the first time, you need to wait tens of seconds because TiKV switches to the data format of Raft Engine from the one of RocksDB.
-> - The data format of Raft Engine in TiDB v5.4.0 is not compatible with previous versions. Therefore, when downgrading a TiDB cluster from v5.4.0 to earlier versions, you need to disable Raft Engine (by setting the value of `enable` to `false`) **before** downgrading and restart TiKV to make the configuration take effect. Otherwise, you cannot start the downgraded cluster.
+>
+> - When you enable Raft Engine for the first time, TiKV transfers its data from RocksDB to Raft Engine. Therefore, you need to wait extra tens of seconds for TiKV to start.
+> - The data format of Raft Engine in TiDB v5.4.0 is not compatible with previous versions. Therefore, if you need to downgrade a TiDB cluster from v5.4.0 to earlier versions, **before** downgrading, disable Raft Engine by setting `enable` to `false` and restarting TiKV for the configuration to take effect.
```
Comment (TomShawn) [inline tikv-configuration-file.md:1324]:
```suggestion
> - The data format of Raft Engine in TiDB v5.4.0 is not compatible with earlier TiDB versions. Therefore, if you need to downgrade a TiDB cluster from v5.4.0 to an earlier version, **before** downgrading, disable Raft Engine by setting `enable` to `false` and restart TiKV for the configuration to take effect.
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

**Pull request merged**